### ISSUE TITLE
Unable to use folders/file present locally from non-package

### ISSUE THREAD

**Issue by nidhi-singh02** [kurtosis-tech/kurtosis#2524]
### Background & motivation

Currently `upload_files` allows to pass the GitHub URL, I have a usecase where I wish to upload files from my local machine into kurtosis enclave. The files are present in the same repository where the kurtosis package is present, but not within the package. 
I went through the other issues that are raised, couldn't find how to do it. Let me know if there's a workaround for this. Can we have this feature in the roadmap pls.

https://github.com/kurtosis-tech/kurtosis/issues/1700 has been closed with the GitHub PR, seems local folder stuff is not yet supported.

### Desired behaviour

Should be able to specify any local file path & turn into a files artifact to use for the services inside their enclaves, without needing to have the desired data to be part of a Kurtosis package.

### How important is this to you?

Critical; Kurtosis is unusable for me without it.

### What area of the product does this pertain to?

CLI: the Command Line Interface

**Comment by tedim52** [comment]
Hey @nidhi-singh02 ! Thanks for creating this issue, the way Kurtosis is designed currently, all assets that need to be referenced by locators (eg. in `upload_files` or `read_file`) need to be placed inside of the Kurtosis package. I would suggest moving any assets you need in the enclave inside the package but maybe there is a reason you can't do this?

**Comment by nidhi-singh02** [comment]
Yes, I understand that. The thing is I have a usecase where I wish to upload files from my local machine into kurtosis enclave. The files are present in the same repository where the kurtosis package is present, but not within the package.
